scripts/deploy-kubefed.sh

```diff
@@ -14,33 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This script automates deployment of a federation - as documented in
-# the README - to the current kubectl context.  It also joins the
-# hosting cluster as a member of the federation.
+# This script automates deployment of a KubeFed control plane - as
+# documented in the README - to the current kubectl context.  It also
```

Comment:
`README` seems not documented control plane. It makes no sense for explaination here. Shall we delete "- as  documented in the README - "

It is not related to KubeFed changes. But it is subtle to do another PR.

Reply:
Fair point.  The README used to document a manual deployment but only helm is supported now.  I'll fix.

Reply:
Done